sunkit_image/tests/test_stara.py

```diff
@@ -0,0 +1,21 @@
+import numpy as np
+
+from sunkit_image.stara import stara
+from sunkit_image.tests.helpers import figure_test
+
+
+def test_stara(mock_hmi_map):
+    result = stara(mock_hmi_map)
+    assert isinstance(result, np.ndarray)
+    assert result.shape == mock_hmi_map.data.shape
```

Comment:
Is there anything else we can check? Values or the locations?

Reply:
The issue is the `result` contains a lot of boolean values, and these value are spread across pixels, I'll have to write a test that checks only the ones that are true.